pepper-apis/studybuilder-cli/studies/singular/patches/singular-race-question-prompt-fix.conf

```diff
@@ -0,0 +1,3 @@
+{
+  "prompt": "What is $ddp.checkAnswer(\"PATIENT_SURVEY_WHO_ENROLLING_HIDDEN\",\"CHILD\", \"your child's\", \"your\") race?*"
```

Comment:
Are we still required to manually add a `*` after required questions or is the front-end handling that?

Reply:
I personally added the Angular code to add the asterisk years ago to required questions. I don't know what happened to it! But you see the screenshots (it was already in the original string we are editing).

Reply:
I see what's going on here-

The question has a `required` validation, but the template is defined as
```
"""$tests_prompt_p1 <span class="Italic">$tests_prompt_p2</span>"""
```

So inserting a `*` at the end of the "prompt" would not result in the correct behavior.